CPAC/pipeline/cpac_pipeline.py

```diff
@@ -744,13 +751,20 @@ def build_anat_preproc_stack(rpool, cfg, pipeline_blocks=None):
                     acpc_align_brain_with_mask
                     # outputs space-T1w_desc-brain_mask for later - keep the mask (the user provided)
                 ]
+                if 'FreeSurfer-BET-Tight' in cfg.anatomical_preproc[
+                    'anatomical_preproc']['brain_extraction']['using'] or 'FreeSurfer-BET-Loose' \
+                        in cfg.anatomical_preproc['anatomical_preproc']['brain_extraction']['using']:
```

Comment:
Isn't this switch automatically handled by the node block/connector?

Reply:
You're right! Resolved